packages/js/data/src/crud/resolvers.ts

```diff
@@ -57,6 +57,13 @@ export const createResolvers = ( {
 		const urlParameters = getUrlParameters( namespace, query || {} );
 		const resourceQuery = cleanQuery( query || {}, namespace );
 
+		yield controls.dispatch(
+			storeName,
+			'startResolution',
+			`get${ pluralResourceName }TotalCount`,
+			[ query ]
+		);
```

Comment:
Make sure we also dispatch a `startResolution` for the total count, given we are also triggering a finished resolution already.